libs/geography/src/reducers/geography/geography.reducer.ts

```diff
@@ -13,6 +13,12 @@ export function daffGeographyReducer<T extends DaffCountry>(
       return { ...state, loading: true };
 
     case DaffGeographyActionTypes.CountryLoadSuccessAction:
+      return {
```

Comment:
It looks like this change is unneeded

Reply:
yeah it was leftover from a previous implementation, good catch. removed.